packages/web-react/src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { AccordionOpenStateType, AccordionProps, AccordionContextProps } from '../../types';
+import { AccordionProvider } from './AccordionContext';
+import { useAccordionStyleProps } from './useAccordionStyleProps';
+
+const Accordion: React.FC<AccordionProps> = (props) => {
+  const { children, elementType: ElementTag = 'section', id, open = [], toggle, ...rest } = props;
+  const { classProps } = useAccordionStyleProps();
+
+  const contextValue: AccordionContextProps = {
+    open,
+    toggle,
+    isItemOpen: (state: AccordionOpenStateType, iid: string) => iid === state || state?.includes(iid),
+  };
+
+  return (
+    <ElementTag className={classProps.root} id={id} {...rest}>
```

Comment:
Missing `styleProps` here.

Reply:
So if the `.Accordion` class doesn't contain any style definition, I decided not to put it there either.

Reply:
Maybe you missed my point, `useStyleProps` hook is the one that allows usage of `UNSAFE_` style props. E. g. now when I use `<Accordion className="Unicorn" />` I will override our class. And also I do not want to allow the usage of `className`.

Reply:
done